cli.js

```diff
@@ -175,7 +175,7 @@ cli.main(function(args, options) {
     if (minified !== null) {
       // Write the output
       try {
-        if (output !== null) {
+        if (output !== null && output !== undefined) {
```

Comment:
Okay. I wasn't sure because of the [.jshintrc](https://github.com/kangax/html-minifier/blob/gh-pages/.jshintrc).